Route sync & k8s iptables mark fix

leodotcloud opened the pull request.

leodotcloud committed:
Moving route add logic from startup script to go code

leodotcloud committed:
Fix to allow k8s traffic in CATTLE_FORWARD change

ibuildthecloud commented on routesync/watcher.go:
```diff
@@ -0,0 +1,115 @@
+func Watch(syncIntervalStr string) error {
+	logrus.Debugf("routesynce: syncIntervalStr: %v", syncIntervalStr)
+
+	var syncInterval int
```
```
syncInterval := DefaultSyncInterval
    if i, err := strconv.Atoi(syncIntervalStr); err != nil {
        syncInterval = i 
    }   
```

ibuildthecloud commented on routesync/watcher.go:
```diff
@@ -0,0 +1,115 @@
+}
+
+func addRouteToMetadataIP(bridgeName, metadataIP string) error {
+	logrus.Debugf("routesynce: adding route to metadata IP address")
```
typo `e`

leodotcloud committed:
Taking care of comments
- refactor code to avoid too much indented code
- typo correction
- add comments
